democracy/views/hearing.py

```diff
@@ -68,6 +69,34 @@ def __init__(self, *args, **kwargs):
         super(HearingCreateUpdateSerializer, self).__init__(*args, **kwargs)
         self.partial = kwargs.get('partial', False)
 
+    def validate(self, data):
+        data = super(HearingCreateUpdateSerializer, self).validate(data)
+        action = self.context['view'].action
+        if self._validate_title_present(action, data):
+            return data
+        raise serializers.ValidationError()
```

Comment:
Is this row ever reached? I guess the above `if` always returns `True` or raises `ValidationError`.